resources/services/network/public_ip_addresses.go

```diff
@@ -27,14 +27,14 @@ func NetworkPublicIPAddresses() *schema.Table {
 				Resolver:    client.ResolveAzureSubscription,
 			},
 			{
-				Name:          "extended_location_name.",
+				Name:          "extended_location_name",
```

Comment:
wouldn't this require a rename on the column?

Reply:
https://github.com/cloudquery/cq-provider-azure/pull/114 introduced this bug, docs and 2 column names have `.` added to them, but no migrations etc.